NaturalSelectedCards/Utils/CookieHelper.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using Microsoft.AspNetCore.Http;
+
+namespace NaturalSelectedCards.Utils
+{
+    public static class CookieHelper
+    {
+        public static CookieOptions GetSecureCookieOptions(int expiresInSeconds) => new CookieOptions
+        {
+            HttpOnly = true,
+            Secure = true,
+            Expires = DateTimeOffset.UtcNow.AddHours(5).AddSeconds(expiresInSeconds)
```

Comment:
Я прочитал, что это время учитывает зону при отправке => мы похерим реальное время жизни токена. 

```suggestion
            Expires = DateTimeOffset.Now.AddSeconds(expiresInSeconds)
```